scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -85,6 +93,22 @@ def process_response(
 
         return response
 
+    @property
+    def _raise_unsupported(self) -> Tuple[bytes]:
+        return (b"br",)
+
+    def _raise_unsupported_compressors(self, request: Request):
+        encodings = request.headers.getlist("Accept-Encoding")
+        if encodings and len(encodings):
+            encodings = encodings.pop().split(ENCODINGS_DELIMETER)
```

Comment:
Not sure why is this doing `encodings.pop()` as you could use `encodings[1]`, but also this should iterate over `encodings` instead of taking the first item.